org.librepcb.LibrePCB.json

```diff
@@ -15,14 +15,14 @@
     "modules": [
         {
             "name": "librepcb",
-            "buildsystem": "qmake",
+            "buildsystem": "cmake",
```

Comment:
we usually use `cmake-ninja`, it's faster.

Reply:
Thanks for the hint! Switched to Ninja in #10.